src/pickley/bstrap.py

```diff
@@ -389,8 +370,9 @@ def main(args=None):
 
     elif package_manager == "uv":
         uv_path = bstrap.find_uv()
-        env = uv_env(mirror=args.mirror, venv=pickley_venv)
-        run_program(uv_path, "-q", "venv", "-p", sys.executable, pickley_venv, env=uv_env(mirror=args.mirror))
+        run_program(uv_path, "-q", "venv", "-p", sys.executable, pickley_venv)
+        env = dict(os.environ)
```

Comment:
I'm not convinced it's any more readable, but you can also do

```
env = {**os.environ, "VIRTUAL_ENV": pickley_venv}
```

since about 3.5 (PEP 448)